### PR TITLE
Add alternate location for first run message

### PR DESCRIPTION
**Pull request by Chuxel**
Adds location that allows codespaces (or anything else) to inject dynamic messages as needed.

### CODE REVIEW

**Review by edgonmsft — APPROVED** [review]
LGTM

**Comment by joshspicer** [comment]
@anpaskin we could potentially use this for an CxC info message

**Review by joshspicer — COMMENTED** [review]
This requires a first-run message be baked into the image right?  There wouldn't be a way to supply it via a devcontainer at codespace build-time right?

File: containers/codespaces-linux/.devcontainer/library-scripts/common-debian.sh [hunk 1]
```diff
@@ -213,9 +213,13 @@ if [ -z "${USER}" ]; then export USER=$(whoami); fi
 if [[ "${PATH}" != *"$HOME/.local/bin"* ]]; then export PATH="${PATH}:$HOME/.local/bin"; fi
 
 # Display optional first run image specific notice if configured and terminal is interactive
-if [ -t 1 ] && [ -f "/usr/local/etc/vscode-dev-containers/first-run-notice.txt" ] && [ ! -f "$HOME/.config/vscode-dev-containers/first-run-notice-already-displayed" ]; then
-    cat /usr/local/etc/vscode-dev-containers/first-run-notice.txt
-    mkdir -p $HOME/.config/vscode-dev-containers
+if [ -t 1 ] && [ ! -f "$HOME/.config/vscode-dev-containers/first-run-notice-already-displayed" ]; then
```
[the remark this answers is not in the document]
Comment (Chuxel) [reply containers/codespaces-linux/.devcontainer/library-scripts/common-debian.sh:216]: Not to my knowledge. This is the best way to detect a terminal I'm aware of. Things like login and interactive shells do not specifically indicate the presence of a terminal, which is what we want in this case.

[the remark this answers is not in the document]
Comment (Chuxel) [reply containers/codespaces-linux/.devcontainer/library-scripts/common-debian.sh:216]: Oh - and the build time is at `/workspaces/.codespaces/shared`. It uses the one in the image, and if this is not present, falls back to whatever is in that location - which is not populated by the image itself.

**Pull request merged**